### PR TITLE
add multiple run for kernel performance testing

### PR DESCRIPTION
**Pull request by JoyceNa**
add multiple run for kernel performance testing

- Related ticket: N/A
- Needles: N/A
- Verification run: http://10.67.132.138/tests/2273

### CODE REVIEW

File: tests/kernel_performance/run_perf_case.pm [hunk 1]
```diff
@@ -33,26 +37,36 @@ sub run_one_by_one {
...
+        }
+        if ($i == $repeat) {
+            power_action('poweroff');
         }
-        if ($time_out == 0) {
-            die "Test run didn't finish within time limit";
+        else {
+            power_action('reboot', textmode => 1, keepconsole => 1);
+            switch_from_ssh_to_sol_console(reset_console_flag => 'on');
+            check_screen('login_screen', 300);
```
Comment (okurz) [inline tests/kernel_performance/run_perf_case.pm]: You are using `check_screen` without checking the return code. I suggest to just use `assert_screen` here

Comment (JoyceNa) [reply tests/kernel_performance/run_perf_case.pm]: Updated.

**Comment by foursixnine** [comment]
Hi @JoyceNa Could you resolve conflicts so that we can merge?

**Comment by JoyceNa** [comment]
Updated,thanks!

**Comment by foursixnine** [comment]
@JoyceNa Sorry for being picky, but the commit message seems to have been changed, can you please change the commit message again, so that it matches the changes in your code? 

Thanks!

**Comment by JoyceNa** [comment]
Thanks for the reminder,I've changed the massage.

**Pull request merged**